src/workflows/modals/PermissionsModal.tsx

```diff
@@ -166,7 +166,7 @@ const CurrentUsers = (props: CurrentUsersProps) => {
 };
 
 export const PermissionsModal = (props: WorkflowPermissionsModalProps) => {
-  const { versionOrNamespace, namespace, setPermissionsModalOpen, refresh, permissionsProvider } = props;
+  const { versionOrCollection, namespace, setPermissionsModalOpen, refresh, permissionsProvider } = props;
```

Comment:
how hard would it be to also change namespace to collection here?

Reply:
I don't this would be hard, but if we do decide to change variable names in this file it would be good to change it in other files as well to stay consistent, and that would not be trivial.